### PR TITLE
helper for getting the extension of a path

### PR DESCRIPTION
**Pull request by dlockhart**
@jstefaniuk-d2l: seem reasonable?

### CODE REVIEW

File: src/getExtension.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+function getExtension(path) {
+
+	if (path === undefined || path === null || typeof(path) !== 'string') {
+		return '';
+	}
+
+	var extension = path.substr(path.lastIndexOf('.') + 1).toLowerCase();
```
Comment (dougmoscrop) [inline src/getExtension.js]: https://www.npmjs.com/package/file-extension
?

File: src/getExtension.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+function getExtension(path) {
+
+	if (path === undefined || path === null || typeof(path) !== 'string') {
```
Comment (dougmoscrop) [inline src/getExtension.js]: the first 2 checks are superflous.. `typeof path  !== 'string'` covers it.

Comment (dlockhart) [inline src/getExtension.js]: Yep, thanks!

File: src/getExtension.js (hunk 1 above)
Comment (dlockhart) [inline src/getExtension.js]: Yeah... agreed. Is it worth it to use something 3rd party for 3 lines of code? We'd have to attribute it in the monolith (at least an hour of work). :-1:

**Comment by jstefaniuk-d2l** [comment]
LGTM

**Pull request merged**